src/middlewares/checkToken.js

```diff
@@ -0,0 +1,25 @@
+import JWThelper from '../utils/jwt';
+import db from '../models';
+import Responses from '../utils/response';
+
+/**
+ * function - decodes the token from the email verification
+ * @param {*} req
+ * @param {*} res
+ * @param {*} next
+ * @returns {undefined}
+ */
+const checkToken = async (req, res, next) => {
+  const { token } = req.query;
```

Comment:
This is okay for this particular endpoint but for other routes, I think we should pass the token in the 'Bearer Token Authorization Header'.